File: applications/account-rest-service/src/main/java/com/vmware/spring/geode/showcase/account/repostories/AccountRepository.java
package com.vmware.spring.geode.showcase.account.repostories;

import com.vmware.spring.geode.showcase.account.domain.account.Account;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AccountRepository
extends CrudRepository<Account,String>
{
}

File: applications/account-jdbc-caching-rest-service/src/main/kotlin/com/vmware/spring/geode/showcase/account/repository/AccountRepository.kt
package com.vmware.spring.geode.showcase.account.repository

import com.vmware.spring.geode.showcase.account.domain.account.Account
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface AccountRepository {

    fun save(account: Account): Account

    fun findById(id: String): Optional<Account>
}